app/src/main/java/de/danoeh/antennapod/activity/MainActivity.java

```diff
@@ -265,6 +266,9 @@ public void loadFragment(String tag, Bundle args) {
             case SubscriptionFragment.TAG:
                 fragment = new SubscriptionFragment();
                 break;
+            case NotesListFragment.TAG:
```

Comment:
Instead of an entry in the main drawer menu (which already is pretty long), how about a filter or a new tab on the episodes screen that shows all episodes that have a note?

Reply:
Thats also a good idea, but for me Notes was THE most important thing, hence I tried to make it easy to access.
Not sure about a "Filter" in episodes. A 4th tab in Episodes (after NEW | ALL | FAVORITES) may not quite gel in that screen, since existing 3 tabs are all directly related to an episode and "Notes" tab may look odd. Thats my personal feeling.

Reply:
> but for me Notes was THE most important thing

We always have to find a balance between adding new features and not overwhelming users who don't use the feature. I think for average users, notes are not the most important thing. 

I agree that more tabs there are not great. See also the discussion in #5075. I think a filter on the "all episodes" screen that shows episodes with notes, while not being perfect for your work-flow, would be better for the average users.